### PR TITLE
Formatting sensors/actuators directories and adding necessary files. Also updating BUILD bazel files in /sensors and /actuators.

### PR DESCRIPTION
**Pull request by bwz5**

### CODE REVIEW

**Comment by bwz5** [comment]
* **#55** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/55?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bwz5 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: actuators/BUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+cc_library(
+    name = "actuator",
+    hdrs = ["actuator.h"],
+    deps = [],
+    visibility = [
+        "//fill:__subpackages__",
+        "//ground:__subpackages__",
```
Comment (ZachGarcia42) [inline actuators/BUILD:7]: We should delete all references to //ground now that that directory is gone

File: fill/telemetry_reader.cc [hunk 2]
```diff
@@ -7,4 +7,4 @@ Telemetry TelemetryReader::read() {
     // TODO(Zach) add telemetry parameters
     telem.set_temp(sensor_.Read());
     return telem;
-}
\ No newline at end of file
+}
```
Comment (ZachGarcia42) [inline fill/telemetry_reader.cc:10]: Please revert this extraneous space for cleanliness

File: sensors/BUILD [hunk 3]
```diff
@@ -9,9 +9,20 @@ cc_library(
...
-    name = "ducer",
-    srcs = ["ducer.cc"],
-    hdrs = ["ducer.h"],
+    name = "load_cell",
+    srcs = ["load_cell.cc"],
+    hdrs = ["load_cell.h"],
+    deps = [":sensor"],
+    visibility = [
+        "//fill:__subpackages__",
+        "//ground:__subpackages__",
```
Comment (ZachGarcia42) [inline sensors/BUILD:18]: more //ground references here

File: sensors/ptd.cc [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#include "ducer.h"
+#include "ptd.h"
 
 float Ducer::Read() {
```
Comment (ZachGarcia42) [inline sensors/ptd.cc:3]: Should also change Ducer::Read if we're making this a PTD library

**Review by ZachGarcia42 — CHANGES_REQUESTED** [review]
Added some comments, but looks really good overall!

File: fill/telemetry_reader.cc (hunk 2 above)
Comment (ZachGarcia42) [reply fill/telemetry_reader.cc:10]: Still see this space

**Review by ZachGarcia42 — APPROVED** [review]
Thanks for the work!

**Comment by bwz5** [comment]
### Merge activity

* **Sep 26, 1:58 PM EDT**: @bwz5 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/55).

**Pull request merged**